CONTRIBUTING.md

```diff
@@ -75,8 +75,22 @@ In order to maintain a high software quality standard, we strongly prefer contri
 
 ## Coding Conventions
 
+Here are some rules of thumb in no particular order:
+
+- Do not use wildcard imports (with a few [exceptions](https://github.com/oss-review-toolkit/ort/blob/master/.detekt.yml)).
+- Sort references (like imports, dependencies etc.) strictly alphabetically (ASCII order, i.e. capital letters first),
+  with blank lines between imports from different top-level packages.
+- Group code logically using blank lines, e.g. before and after code blocks enclosed by curly braces, or before return
+  statements.
+- Do not use blank lines directly after opening curly braces.
+- Do not use duplicate blank lines.
+- Use proper spelling, grammar, and punctuation in *all* documentation, code comments (incl. TODOs) and commit messages.
+- Do not commit out-commented code.
```

Comment:
I observed in recent review also that we want non-cryptic variables names, e.g. no abbrevations if they are not crystal clear. And there would be exception to this rule like always.

Reply:
This is also something I'd like to keep as a future improvement, as at least I wasn't having that issue on a regular basis yet.

Reply:
I wasn't having this on a regular basis either, but I observed it for new contributors.